docs/en/04_Changelogs/4.0.0.md

```diff
@@ -1105,10 +1255,45 @@ These methods are deprecated:
 #### Implementation of ownership API
 
 In order to support the recursive publishing of dataobjects, a new API has been developed to allow
-developers to declare dependencies between objects. See the
-[versioned documentation](/developer_guides/model/versioning) for more information.
+developers to declare dependencies between objects. This is done to ensure that the published state
+of linked components are consistent with their "owner." Without the concept of ownership, these linked
+components could be implicitly exposed on the frontend, which may not align with the intent of the
+content author.
 
-By default all versioned dataobjects will automatically publish objects that they own.
+For instance, on a products page which has a list of products, the products should not be published or
```

Comment:
"published **or unpublished**" is incorrect. We're still discussing how cascading unpublish would work, so the ownership API doesn't "solve this" at the moment.